SwaprHQ/stackly-ui
# [SWA-199][INFRA] - Track click events on Strategies

## Fixes: [SWA-199](https://linear.app/swaprhq/issue/STK-199/track-click-events-on-strategies)

# Description
* Prune unused files and code
* Update env vars and re-arrange constants, types and contexts where they belong
* Update FA existing context and apply an event name to Strategy Cards

# Visual Evidence
https://app.usefathom.com/?range=last_30_days&site=XJTHZCQS -> this is the link to Stackly dapp (dev env) site analytics

Screenshot: 
![image](https://github.com/SwaprHQ/stackly-ui/assets/21271189/a5c87348-b797-4447-b041-819790ddf3ac)


# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Stackly landing page
4) Connect your wallet
5) Clikcing a Strategy Card should trigger a FA event
